<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1558526346401' stop='1558526379766'>
    <name>Automating Bank Manager Login Functionality</name>
    <title>Automating Bank Manager Login Functionality</title>
    <test-cases>
        <test-case start='1558526346402' status='passed' stop='1558526354536'>
            <name>Login to the Account</name>
            <title>Login to the Account</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='e1787f7c-0659-45d8-b37d-213bb3f46d2e-attachment.png' type='image/png' size='12981'/>
            </attachments>
        </test-case>
        <test-case start='1558526354537' status='failed' stop='1558526358038'>
            <name>Validate Open Account</name>
            <title>Validate Open Account</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='21e83087-f2ff-4011-a1fd-003b55a8837b-attachment.png' type='image/png' size='20393'/>
            </attachments>
            <failure>
                <message>Failed: No element found using locator: by.model("fName")</message>
                <stack-trace>NoSuchElementError: No element found using locator: by.model("fName")
                    at elementArrayFinder.getWebElements.then (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
                    at ManagedPromise.invokeCallback_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at asyncRun (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
                    at C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at process._tickCallback (internal/process/next_tick.js:68:7)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as sendKeys] (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.(anonymous function).args [as sendKeys] (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at UserContext.&lt;anonymous> (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\AutomatingBankerLogin\BankManagerLogin.js:21:36)
                    at C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                From: Task: Run it("Validate Open Account") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\AutomatingBankerLogin\BankManagerLogin.js:17:5)
                    at addSpecsToSuite (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\AutomatingBankerLogin\BankManagerLogin.js:6:1)
                    at Module._compile (internal/modules/cjs/loader.js:688:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
                    at Module.load (internal/modules/cjs/loader.js:598:32)
                    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1558526358038' status='failed' stop='1558526379765'>
            <name>Validate Add Customer</name>
            <title>Validate Add Customer</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: No element found using locator: By(css selector, *[id="userSelect"])</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(css selector, *[id="userSelect"])
                    at elementArrayFinder.getWebElements.then (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
                    at ManagedPromise.invokeCallback_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at asyncRun (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
                    at C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at process._tickCallback (internal/process/next_tick.js:68:7)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at SelectWrapper.selectByText (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\select-wrapper.js:20:70)
                    at UserContext.&lt;anonymous> (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\AutomatingBankerLogin\BankManagerLogin.js:48:18)
                    at C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                From: Task: Run it("Validate Add Customer") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\AutomatingBankerLogin\BankManagerLogin.js:44:5)
                    at addSpecsToSuite (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Harish\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Harish\Desktop\Automation\Protractor\E2E Banking Application\AutomatingBankerLogin\BankManagerLogin.js:6:1)
                    at Module._compile (internal/modules/cjs/loader.js:688:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
                    at Module.load (internal/modules/cjs/loader.js:598:32)
                    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>